lib/Doctrine/DBAL/Connection.php

```diff
@@ -773,7 +774,7 @@ public function prepare($statement)
     {
         try {
             $stmt = new Statement($statement, $this);
-        } catch (\Exception $ex) {
+        } catch (Exception $ex) {
```

Comment:
I changed this because it was incosistent in this file (didn't look at the other files tho).